test/DynamoCoreTests/NodeToCodeTest.cs

```diff
@@ -1191,6 +1191,36 @@ public void TestNodeToCodeUndoRecorder()
             Assert.AreEqual(0, recorder.ActionCount());
         }
 
+        [Test]
+        public void TestNodeToCodeStringInputEscaping()
+        {
+            OpenModel(@"core\node2code\stringNodesInNeedOfEscaping.dyn");
+            var nodes = CurrentDynamoModel.CurrentWorkspace.Nodes;
+            var engine = CurrentDynamoModel.EngineController;
+            var result = engine.ConvertNodesToCode(nodes, nodes);
+            Assert.IsNotNull(result.AstNodes);
+
+            var assignment = result.AstNodes.FirstOrDefault();
+            Assert.IsNotNull(assignment);
+
+            var binaryExpr = assignment as BinaryExpressionNode;
+            Assert.IsNotNull(binaryExpr);
```

Comment:
Is all of this actually needed and if it is what are we using it for? 

Shouldnt we just test that the nodes preview value is the expected?

Reply:
I looked into this further and the other NodeToCode tests are extracting the the `BinaryExpressionNode` and checking the `RightNode` in order to check the value of the CodeBlockNode itself. As this seems to be a more explicit way of checking that that NodeToCode has worked as expected I've revised the test to follow this methodology in 3e11fc3.